CardanoSharp.Wallet/Models/Addresses/Address.cs

```diff
@@ -21,6 +23,8 @@ public Address(string prefix, byte[] address)
         {
             _bytes = address;
             _address = Bech32.Encode(address, prefix);
+
+            IsValid = Bech32.IsValid(_address);
```

Comment:
just curious, should we rather throw an exception here or is having and populating the isvalid field more inline with the general style or is it part of a cddl or something?

Reply:
my thought was, i don't want to prevent the construction of the object just because its an invalid address... its still an address that you might still need to process. if we do go the exception route, i recommend a specific exception like `InvalidAddressException` or something.